### PR TITLE
Refactor and fix violations of the ruby style guide

### PR DESCRIPTION
**Pull request by cstavitsky**
Was unsure about the recite_facts method from line 24 to 34, would be interested to know if you think this method should be broken up at all.

### CODE REVIEW

File: triangle_facts/triangle_facts.rb [hunk 1]
```diff
@@ -1,49 +1,48 @@
...
-	def isosceles()
-		return [side1,side2,side3].uniq.length == 2
+	def isosceles?
+		[side_1, side_2, side_3].uniq.length == 2
 	end
 
-	def scalene()
-		unless ! (equilateral || isosceles)
+	def scalene?
+		return true unless (equilateral? || isosceles?)
```
Comment (zsyed1) [inline triangle_facts/triangle_facts.rb]: Lines 20 - 21: You can squash this into one line - 

```
def scalene?
  !equilateral && !isosceles?
end
```

File: triangle_facts/triangle_facts.rb [hunk 2]
```diff
@@ -1,49 +1,48 @@
...
 
 		puts 'This triangle is also a right triangle!' if angles.include? 90
 		puts ''
 	end
 
-	def calculate_angles(a,b,c)
-		angleA = radians_to_degrees(Math.acos((b**2 + c**2 - a**2) / (2.0 * b * c)))
-		angleB = radians_to_degrees(Math.acos((a**2 + c**2 - b**2) / (2.0 * a * c)))
-		angleC = radians_to_degrees(Math.acos((a**2 + b**2 - c**2) / (2.0 * a * b)))
+	def calculate_angles(side_1, side_2, side_3)
```
Comment (zsyed1) [inline triangle_facts/triangle_facts.rb]: Lines 36-42: Why not just make a single method on the class:
```
def angles
...
end
```
Then you can remove line 29 entirely.

File: triangle_facts/triangle_facts.rb [hunk 3]
```diff
@@ -1,49 +1,48 @@
...
+		angle_A = radians_to_degrees(Math.acos((side_2**2 + side_3**2 - side_1**2) / (2.0 * side_2 * side_3)))
+		angle_B = radians_to_degrees(Math.acos((side_1**2 + side_3**2 - side_2**2) / (2.0 * side_1 * side_3)))
+		angle_C = radians_to_degrees(Math.acos((side_1**2 + side_2**2 - side_3**2) / (2.0 * side_1 * side_2)))
 
-		return [angleA, angleB, angleC]
+		[angle_A, angle_B, angle_C]
```
Comment (zsyed1) [inline triangle_facts/triangle_facts.rb]: Lines 37 - 39 are too repetitive. Create a single method:
```
def included_angle(a, b, c)
  angle = Math.acos((a**2 + b**2 - c**2) / (2.0 * a * b))
  radians_to_degrees(angle)
end
```

Then invoke this method three times in `angles`:
```
def angles
  [included_angle(a, b, c),
   included_angle(a, c, b),
   included_angle(b, c, a)]
end
```

File: triangle_facts/triangle_facts.rb [hunk 4]
```diff
@@ -1,49 +1,48 @@
...
-		puts 'This triangle is scalene and mathematically boring.' if scalene 
+		puts 'This triangle is equilateral!' if equilateral?
+		puts 'This triangle is isosceles! Also, that word is hard to type.' if isosceles? 
+		puts 'This triangle is scalene and mathematically boring.' if scalene?
 
-		angles = self.calculate_angles(side1,side2,side3)
+		angles = self.calculate_angles(side_1, side_2, side_3)
 		puts 'The angles of this triangle are ' + angles.join(',')
 
 		puts 'This triangle is also a right triangle!' if angles.include? 90
```
Comment (zsyed1) [inline triangle_facts/triangle_facts.rb]: Why not move the 90 stuff into a method like scalene and the others:

```
def right?
   angles.include?(90)
end
```

Also, put parentheses around method arguments (except for `puts`)

File: triangle_facts/triangle_facts.rb [hunk 5]
```diff
@@ -1,49 +1,48 @@
...
-	  end
 	end
 
 	def recite_facts
-		puts 'This triangle is equilateral!' if equilateral
-		puts 'This triangle is isosceles! Also, that word is hard to type.' if isosceles 
-		puts 'This triangle is scalene and mathematically boring.' if scalene 
+		puts 'This triangle is equilateral!' if equilateral?
+		puts 'This triangle is isosceles! Also, that word is hard to type.' if isosceles? 
+		puts 'This triangle is scalene and mathematically boring.' if scalene?
```
Comment (zsyed1) [inline triangle_facts/triangle_facts.rb]: You can remove unnecessary blank lines.

File: triangle_facts/triangle_facts.rb [hunk 6]
```diff
@@ -1,49 +1,48 @@
...
 
-		angles = self.calculate_angles(side1,side2,side3)
+		angles = self.calculate_angles(side_1, side_2, side_3)
 		puts 'The angles of this triangle are ' + angles.join(',')
```
Comment (zsyed1) [inline triangle_facts/triangle_facts.rb]: Line 30: Use interpolation to display strings:

```
puts "The angles of this triangle are #{angles.join(',')}"
```

File: triangle_facts/triangle_facts.rb [hunk 7]
```diff
@@ -1,49 +1,48 @@
...
 
-		return [angleA, angleB, angleC]
+		[angle_A, angle_B, angle_C]
 	end
 
 	def radians_to_degrees(rads)
-		return (rads * 180 / Math::PI).round
+		(rads * 180 / Math::PI).round
 	end
 end
```
Comment (zsyed1) [inline triangle_facts/triangle_facts.rb]: Nice work. The changes I've suggested are all style-things that you'll have to get used to when working in the Ruby community. For that purpose alone I would recommend making the changes. It's up to Sinsei Haddad whether you need to resubmit! @paulghaddad

File: triangle_facts/triangle_facts.rb [hunk 8]
```diff
@@ -1,49 +1,48 @@
 # Killer facts about triangles AWW YEAH
 class Triangle
-	attr_accessor :side1,:side2,:side3
+	attr_accessor :side_1, :side_2, :side_3
 
-	def initialize(side1,side2,side3)
-		@side1,@side2,@side3 = side1,side2,side3
+	def initialize(side_1, side_2, side_3)
+		@side_1 = side_1
```
Comment (paulghaddad) [inline triangle_facts/triangle_facts.rb]: It appears that your indentation is too much. Two spaces is preferred.

File: triangle_facts/triangle_facts.rb (hunk 2 above)
Comment (paulghaddad) [inline triangle_facts/triangle_facts.rb]: Try to make methods `private` if they are called from outside the class.

**Comment by paulghaddad** [comment]
@cstavitsky This looks good. Take a look at our comments and do a quick resubmission. Also note the indentation.

As discussed in Hipchat, please title the next PR as "Triangle Facts - Iteration 1".

**Pull request closed without merging**